src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/RazorDiagnosticsEndpoint.cs

```diff
@@ -99,6 +112,8 @@ public async Task<RazorDiagnosticsResponse> Handle(RazorDiagnosticsParams reques
             var filteredDiagnostics = unmappedDiagnostics.Where(d => !CanDiagnosticBeFiltered(request.Kind, d)).ToArray();
             if (!filteredDiagnostics.Any())
             {
+                _logger.LogInformation("No diagnostics remaining after filtering.");
+
                 // No diagnostics left after filtering.
```

Comment:
Nitpick: The log message kinda takes care of this comments job now.

Reply:
I'll address this in another PR so that it doesn't have to go through CI again.